sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -135,13 +135,14 @@ public:
   bfloat16(const float &a);
   bfloat16 &operator=(const float &a);
 
-  // Convert from bfloat16 to float
+  // Convert bfloat16 to floating-point types
   operator float() const;
+  operator sycl::half() const;
 
   // Get bfloat16 as uint16.
   operator storage_t() const;
```

Comment:
Somehow this conversion function reappeared after your merge.

Reply:
I think this was just a merge mistake, so I removed it in 37835ee9d094969b7d3516da3b8f2395b98b451c